registry.json

```diff
@@ -859,6 +859,30 @@
       "documentation": "https://polkadot.js.org/api/substrate/rpc.html"
     }
   },
+  {
+    "id": "astar",
+    "name": "Astar",
+    "coinId": 634,
+    "symbol": "ASTR",
+    "decimals": 18,
+    "blockchain": "Polkadot",
+    "derivationPath": "m/44'/434'/0'/0'/0'",
```

Comment:
Is 634 the correct coin ID?  In derivation path 434 is used.  Astar is not found in slip-44, and 434 is the Id of Kusama.

Reply:
Yes, you are right, let me fix it

Reply:
Waiting for merge https://github.com/satoshilabs/slips/pull/1272

Reply:
It got merged, with value `810`.  Can you please consolidate the 3 values? (654, 434, 810).  Ideally one should be sued for coin type, in the derivation path, and slip44.